paasta_tools/cli/cmds/validate.py

```diff
@@ -658,7 +660,10 @@ def validate_min_max_instances(service_path):
                 load_deployments=False,
                 soa_dir=soa_dir,
             )
-            if instance_config.get_instance_type() != "tron":
+            # TODO: Remove this cast. This will require updating utils.LONG_RUNNING_INSTANCE_TYPE_HANDLERS to handle tron and adhoc type
+            # and change get_instance_config to return an object of type LongRunningServiceConfig
```

Comment:
we'd probably want to do something slightly different since tron/adhoc aren't actually long-running so we don't want them to be of type `LongRunningServiceConfig` - i'd say it's probably fine to just leave this as a comment about how we're iterating over `InstanceConfig` objects and only certain instance types subclass `InstanceConfig` to provide autoscaling (and we don't partition these into long-running/not long-running at iteration time)